tests/unit/plugins/modules/test_kubevirt_vm.py

```diff
@@ -132,6 +172,24 @@ def module_params_create(module_params_default):
     }
 
 
+@pytest.fixture(scope="module")
+def module_params_running(module_params_default):
+    return module_params_default | {
+        "name": "testvm",
+        "namespace": "default",
```

Comment:
What about extracting these two to `module_params_default`?

Reply:
Wanted to keep only default value params in `module_params_default`. Also we cannot have `name` and `generate_name` in there at the same time while in k8s_module_params we can.

Reply:
Got it. Thanks for the clarification.